src/loaders/bitmapFontParser.js

```diff
@@ -1,8 +1,64 @@
 var Resource = require('resource-loader').Resource,
     core = require('../core'),
+    utils = require('../core/utils'),
     extras = require('../extras'),
     path = require('path');
 
+
+var parse = function(resource, texture) {
```

Comment:
the expression here is evaluated are runtime, the other is evaluated at parse time. Always prefer the named function unless you need the expression for some reason.